docs/migration/5_0.md

```diff
@@ -0,0 +1,4 @@
+# Technical Changes in Spartacus 5.0
+
+#### MediaComponent
+- `get loadingStrategy()` changed return type from `string` to `ImageLoadingStrategy | null`.
```

Comment:
IMHO we should add a note in migration doc only in the PR which acutally introduces the breaking change. Here we don't.

Btw. it's nicely handled here to avoid breaking changes 😉

Reply:
I created a PR for the deprecations as an experiment to see if we can avoid making tickets to explain the fixes for breaking changes. The main risk I see here is the branch becoming stale. Are there other issues you can think of with this approach?
https://github.com/SAP/spartacus/pull/14236/files